backend/custom_models/user_models.py

```diff
@@ -16,3 +17,15 @@ def get_by_natural_key(self, username):
 
 class User(AbstractUser):
     objects = CustomUserManager()
+    first_name = models.CharField(max_length=30, null=False, blank=False)
+    last_name = models.CharField(max_length=150, null=False, blank=False)
+    birth_date = models.CharField(max_length=10, null=False, blank=False)
+    pri_or_military_nbr = models.CharField(max_length=9, null=True, blank=True)
+    REQUIRED_FIELDS = [
+        "first_name",
+        "last_name",
+        "birth_date",
+        "email",
+        "pri_or_military_nbr",

```

Comment:
just curious, is this one actually required? Since some applications will not be part of the government already, I would think this could be null

Reply:
This _REQUIRED_FIELDS_ setting makes sure that these fields are displayed in the DB. Then, the _null=True_ and _blank=True_ settings set this field as not mandatory when creating a new account using the user model.

That means that all fields that we want mandatory need to be _null=False_ and _blank=False_.

Reply:
ahh, ok.